cli/githubutil.py

```diff
@@ -140,3 +140,22 @@ def release_note_blocks_cli(
         repository_branch=repository_branch,
         commit_range=commit_range
     ).release_note_blocks()
+
+
+def delete_releases(
+    github_cfg_name: str,
+    github_repository_owner: str,
+    github_repository_name: str,
+    release_names: CliHint(type=str, nargs='+'),
```

Comment:
consider making this consistent with our other coding; we use type=[str] elsewhere so far. this would introduce a new cli-style